tests/model_serving/model_server/private_endpoint/constants.py

```diff
@@ -0,0 +1,33 @@
+from typing import Dict, List, Any
+
+AWS_REGION: str = "us-east-2"
+AWS_BUCKET: str = "ods-ci-wisdom"
+AWS_ENDPOINT: str = "https://s3.us-east-2.amazonaws.com/"
+
```

Comment:
may be u can check  old code for pvc how its written

Reply:
Not sure I understand the suggestion